package.json

```diff
@@ -58,8 +58,9 @@
     "preact": "^10.4.0"
   },
   "scripts": {
-    "build-js": "babel src --out-dir lib --source-maps --ignore '**/test' --ignore '**/karma.config.js'",
-    "build": "yarn build-js && tsc --build src/tsconfig.json",
+    "build-esm": "babel src --out-dir lib/ --source-maps --ignore '**/test' --ignore '**/karma.config.js'",
+    "build-cjs": "babel src --out-dir lib-cjs/ --source-maps --ignore '**/test' --ignore '**/karma.config.js' --config-file ./.babelrc-cjs.js",
+    "build": "yarn build-cjs && yarn build-esm && tsc --build src/tsconfig.json",
```

Comment:
Is it desired that the CJS and ESM builds will build even if `tsc` fails? Or might it make more sense to have the `tsc` command first? Thoughts?

Reply:
The most important thing is that CI fails if `tsc` fails. Otherwise I don't think it matters too much either way.